common/app/model/Cached.scala

```diff
@@ -32,6 +32,7 @@ object CacheTime {
   object FootballMatch extends CacheTime(30)
   object Cricket extends CacheTime(60)
   object FootballTables extends CacheTime(60)
+  object DCARAssets extends CacheTime(10)
```

Comment:
I assume the relatively short cache time is so that we pick up fresh changes to the JS bundle?

Reply:
If we want to keep the JS bundle in sync with the latest release, 10mins is OK